doc/nrf/app_dev/device_guides/nrf54l/cryptography.rst

```diff
@@ -0,0 +1,300 @@
+|             |                                        |                                                                                                                        |
+|             |                                        | Encrypted keys require two additional KMU slots to store the authentication nonce and tag.                             |
++-------------+----------------------------------------+------------------------------------------------------------------------------------------------------------------------+
+| Raw         | CRACEN_KMU_KEY_USAGE_SCHEME_RAW        | Keys are stored as plaintext and pushed to a CPU-accessible RAM location before being used by the CRACEN.              |
++-------------+----------------------------------------+------------------------------------------------------------------------------------------------------------------------+
+
+.. _ug_nrf54l_crypto_kmu_supported_key_types:
+
+Supported key types
+-------------------
+
+The table below lists all key types that can be stored in the KMU, in addition to the number of key slots they require and which one(s) of the usage schemes Protected, Encrypted and Raw they are supporting.
+
+.. list-table:: Supported key types
+   :widths: auto
+   :header-rows: 1
+
+   * - Key type
+     - PSA key attributes
+     - KMU slots [1]_
+     - Protected
+     - Encrypted
+     - Raw
+   * - AES 128-bit keys
+     - | key_type: PSA KEY_TYPE_AES
+       | key_bits: 128
+     - 1
+     - Yes
+     - Yes
+     - Yes
+   * - AES 192-bit and 256-bit keys
+     - | key_type: PSA_KEY_TYPE_AES
+       | key_bits: 192
+       | key_bits: 256
+     - 2
+     - Yes
+     - Yes
+     - Yes
+   * - ChaCha20-Poly1305
+     - key_type: PSA_KEY_TYPE_CHACHA20
+     - 2
+     - No
+     - Yes
+     - Yes
+   * - ECC secp256r1 key pair
+     - | key_type: PSA_KEY_TYPE_ECC_KEY_PAIR(PSA_ECC_FAMILY_SECP_R1)
+       | key_bits: 256
+     - 2
+     - No
+     - Yes
+     - Yes
+   * - ECC secp256r1 public key
+     - | key_type: PSA_KEY_TYPE_ECC_PUBLIC_KEY(PSA_ECC_FAMILY_SECP_R1)
+       | key_bits: 256
+     - 4
+     - No
+     - Yes
+     - Yes
+   * - Ed25519 key pair
+     - | key_type: PSA_KEY_TYPE_ECC_KEY_PAIR(PSA_ECC_FAMILY_TWISTED_EDWARDS)
+       | key_bits: 255
+     - 2
+     - No
+     - Yes
+     - Yes
+   * - ED25519 public key
+     - | key_type: PSA_KEY_TYPE_ECC_PUBLIC_KEY(PSA_ECC_FAMILY_TWISTED_EDWARDS)
+       | key bits: 255
+     - 2
+     - No
+     - Yes
+     - Yes
+.. [1] Keys with the Encrypted usage scheme (CRACEN_KMU_KEY_USAGE_SCHEME_ENCRYPTED) will require two additional KMU slots to store the nonce and the authentication tag.
+
+Storing keys in KMU
+-------------------
+
+Applications can store keys in KMU slots using the standard PSA cryptographic key management operations psa_import_key, psa_generate_key or psa_copy_key.
```

Comment:
I know it is another topic, but we should really integrate [PSA Crypto API]( https://arm-software.github.io/psa-api/crypto/1.2/) docs into our docs soon, so we can link these API calls.